shell_automaton/src/service/protocol_runner_service.rs

```diff
@@ -49,6 +50,13 @@ pub enum ProtocolRunnerResult {
     ),
     InitContextIpcServer((ProtocolRunnerToken, Result<(), ProtocolServiceError>)),
 
+    GetCurrentHead(
```

Comment:
Please move this bellow:
```
GenesisCommitResultGet(
        (
            ProtocolRunnerToken,
            Result<CommitGenesisResult, ProtocolServiceError>,
        ),
    ),
```
To write them in same order as they are later called in. `GenesisCommitResultGet` is called before `GetCurrentHead`.

Also please rename it to `CurrentHeadGet` to make naming consistent.

Reply:
Alright, I will rename to `LatestContextHashesGet`